_includes/contact-us/contact-us.scss

```diff
@@ -0,0 +1,138 @@
+@import 'mixins';
+
+.contact-us {
+  background-color: black;
+  padding: 120px 0 136px !important;
+
+  .text-center {
+    margin-bottom: 74px;
+  }
+
+  .form-wrapper {
+    @include flex(column, flex-start, flex-start);
+    flex-wrap: wrap;
+    max-width: 960px;
```

Comment:
I thought we have `content` class for that purpose, or do we have different dimensions here?

Reply:
content have 1200px max width but we need max-width 960px for form